### PR TITLE
Change both Hacker and Countermeasure tanks to Utility tab

### PR DESCRIPTION
**Pull request by WilliamWeberBerrutti**
- Disables both land and naval minelayers
- Moves both Hacker and Countermeasure tanks to Utility tab

### CODE REVIEW

File: mods/hv/rules/vehicles.yaml [hunk 1]
```diff
@@ -831,7 +831,7 @@ HACKERTANK:
 	Inherits: ^TrackedVehicle
 	Inherits@AutoTarget: ^AutoTargetGroundAssaultMove
 	Buildable:
-		Queue: Tank
+		Queue: Utility
 		BuildPaletteOrder: 60
 		Prerequisites: ~factory.yi, radar, trader
```
Comment (Mailaender) [inline mods/hv/rules/vehicles.yaml:836]: This makes them invisible before a factory is built which is irritating.

Comment (WilliamWeberBerrutti) [reply mods/hv/rules/vehicles.yaml:836]: Oh didn't know that! How to fix it?

Comment (OcelotWalrus) [reply mods/hv/rules/vehicles.yaml:836]: Just remove the `~`.

Comment (Mailaender) [reply mods/hv/rules/vehicles.yaml:836]: That will show the internal string `factory.yi` in the build palette.

Comment (OcelotWalrus) [reply mods/hv/rules/vehicles.yaml:836]: Oh yeah right. But anyway, I don't see why they're being moved to the utility tabs, they don't belong there.

**Comment by Dzierzan** [comment]
`Disables both land and naval minelayers`

No, I do not accept that. You don't use them in MP? OK, so don't use them, but AI uses them nicely. So I see no problem here.

**Comment by GraionDilach** [comment]
> Disables both land and naval minelayers

This is already dropped from the PR.

**Comment by WilliamWeberBerrutti** [comment]
Yea this experimental PR was a bad idea. Thanks for the feedbacks! Closing it!

**Pull request closed without merging**